opencti-platform/opencti-graphql/src/database/data-builder.js

```diff
@@ -223,7 +223,7 @@ export const buildRelationData = async (context, user, input, opts = {}) => {
     data.modified = R.isNil(input.modified) ? today : input.modified;
     //* v8 ignore if */
     if (data.start_time > data.stop_time) {
-      throw DatabaseError('You cant create a relation with a start_time less than the stop_time', {
+      throw DatabaseError('You cant create a relation with a stop_time less than the start_time', {
```

Comment:
why this change ? the message does look good right ? stop_time < start_time. Maybe "anterior" instead of "less" ?

Reply:
no stop_time and start_time are not in the good order. It is stop_time that can't be inferior to start_time, not the other way round.

Reply:
we can't have stop_time < start_time

Reply:
oh right, exactly